web/pdf_page_view.js

```diff
@@ -69,6 +71,7 @@ var PDFPageView = (function PDFPageViewClosure() {
     var renderingQueue = options.renderingQueue;
     var textLayerFactory = options.textLayerFactory;
     var annotationLayerFactory = options.annotationLayerFactory;
+    var enhanceTextSelection = options.enhanceTextSelection || false;
```

Comment:
Given the performance/correctness issues currently present, I don't think that we want to enable this by default at this point (so `var ENHANCE_TEXT_SELECTION = false;` should be set in app.js).

More to the point though, please note that we agreed to https://github.com/mozilla/pdf.js/pull/7539#issuecomment-242080291, to avoid changing the defaults for the viewer components. Hence this comment is not applicable.